.travis.yml

```diff
@@ -27,17 +29,30 @@ env:
         - CPPFLAGS="-I$HOME/bogus/include"
         - LDFLAGS="-I$HOME/bogus/lib"
         - LD_LIBRARY_PATH="$HOME/bogus/lib"
-        - CONFIGURE_ARGS="--prefix=$HOME/bogus"
-        - DISTCHECK_CONFIGURE_FLAGS="$CONFIGURE_ARGS"
+    matrix:
+        - GCC_VERSION=default
+        - GCC_VERSION=5
 
-# Install dependencies for the verbs and usnic providers.  Note that
-# the libfabric verbs provider cannot currently build on Travis, so we
-# disable it with --disable-verbs (see
-# https://github.com/ofiwg/libfabric/issues/1553 for a description of
-# the RAI_FAMILY issue).
+# install dependencies for the verbs and usnic providers
+# libfbric verbs provider cannot current build without RAI_FAMILY
+# see https://github.com/ofiwg/libfabric/issues/1553
+# bla
```

Comment:
I thought Travis somehow blacklisted the original pr, so I made a new one
my comment is not related whatsoever to what you wrote above, I simply wanted some change to commit, hopping Travis would start a test.
Travis tested #1228(created a few hours ago) but not this one (!)

I will commit my configure change only tmoor row(I added -I. to FCFLAGS as you pointed), and keep playing with travis in an other pr.
btw, do you want to test 6 environments {linux,osc}*{clang,default gcc,gcc 5} or 4 (e.g. do not test with default gcc since it cannot build f08 bindings)

the error is odd ...
one source file works fine on my mac (yosemite with latest patches, and gcc 5.3 from brew) and centos 7.
but fails on Travis (same gcc 5.3, older xcode, I do not know which osx)
and the test fails with pgi 15.10 with 1 or 3 source files.
so I guess this is not worth reporting, since this issue is solved with recent osx, and I do not know the root cause (gcc ?, OS X ? Xcode ? strange interaction between gcc and old osx/xcode ?)